Lib/ufo2ft/featureWriters/baseFeatureWriter.py

```diff
@@ -5,7 +5,7 @@
 from ufo2ft.errors import InvalidFeaturesData
 from ufo2ft.featureWriters import ast
 
-INSERT_FEATURE_MARKER = r"\s*# Automatic Code"
+INSERT_FEATURE_MARKER = r"\s*# Automatic Code.*"
```

Comment:
how about we only match optional known suffixes `(?: (Start|End))?` instead of too broad `.*`?

Reply:
unless Glyphs.app also supports strings like `# Automatic Code Whatever`, in which case.. whatever

Reply:
Glyphs 3 seems no not care what comes after `# Automatic Code`, and indeed `# Automatic Code Whatever` works.

Reply:
LOL ok then, let's merge as is